tools-integration/src/test/java/com/hedera/hashgraph/client/integration/TestUtil.java

```diff
@@ -439,6 +447,21 @@ public static PasswordField findPasswordInPopup(ObservableList<Node> popupNodes)
 		return null;
 	}
 
+	public static TextField findTextFieldInPopup(ObservableList<Node> popupNodes) {
+		for (Node popupNode : popupNodes) {
+			if (popupNode instanceof TextField) {
+				return (TextField) popupNode;
+			}
+			if (popupNode instanceof HBox) {
+				return findPasswordInPopup(((HBox) popupNode).getChildren());
+			}
+			if (popupNode instanceof VBox) {
+				return findPasswordInPopup(((VBox) popupNode).getChildren());
+			}
```

Comment:
These can be `if -- else if ` conditions

Reply:
made the change